### ISSUE TITLE
high contrast mode - missing vertical line when sidebar collapsed

### ISSUE THREAD

**Issue by mousetraps** [Microsoft/vscode#17136]
- VSCode Version: Code 1.8.0 (38746938a4ab94f2f57d9e1309c51fd6fb37553d, 2016-12-13T17:45:32.595Z)
- OS Version: Windows_NT ia32 10.0.14965

![image](https://cloud.githubusercontent.com/assets/762848/21158780/75d70efe-c133-11e6-829c-0ce57dcec4d0.png)
